src/controller/review.py

```diff
@@ -1,3 +1,37 @@
-from flask import Blueprint
+from flask import Blueprint, Response
+from ..lib import loadCSV
+from ..db.models import ApplicationsModel
+import os
 
 review = Blueprint('review', __name__)
+
+_dirname = os.path.dirname(__file__)
+
+@review.route('/configure/upload_csv')
+def doCSV():
+    with open(os.path.join(_dirname, "../lib/sample.csv"), encoding="utf-8") as data:
+        loadCSV.upload_csv(data)
+    return Response(status=200)
+
+@review.route('/review')
+def getApplicant():
+    model = ApplicationsModel.get_applications_model()
+    try:
+        data = model.getLockedApplicantRowForUser(1)
```

Comment:
Putting a note here that the user is hardcoded since authentication is not done. This should be fixed in a future PR